src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,481 @@
+GRANT INSERT ON "SDT_OWNER"."GLOBAL_PARAMETERS" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."GLOBAL_PARAMETERS" TO "SDT_USER";
+GRANT UPDATE ON "SDT_OWNER"."GLOBAL_PARAMETERS" TO "SDT_USER";
+GRANT DELETE ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" TO "SDT_USER";
+GRANT INSERT ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" TO "SDT_USER";
+GRANT UPDATE ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" TO "SDT_USER";
+GRANT DELETE ON "SDT_OWNER"."SERVICE_ROUTINGS" TO "SDT_USER";
+GRANT INSERT ON "SDT_OWNER"."SERVICE_ROUTINGS" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SERVICE_ROUTINGS" TO "SDT_USER";
+GRANT UPDATE ON "SDT_OWNER"."SERVICE_ROUTINGS" TO "SDT_USER";
+GRANT DELETE ON "SDT_OWNER"."SERVICE_TYPES" TO "SDT_USER";
+GRANT INSERT ON "SDT_OWNER"."SERVICE_TYPES" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SERVICE_TYPES" TO "SDT_USER";
+GRANT UPDATE ON "SDT_OWNER"."SERVICE_TYPES" TO "SDT_USER";
+GRANT DELETE ON "SDT_OWNER"."SERVICE_REQUESTS" TO "SDT_USER";
+GRANT INSERT ON "SDT_OWNER"."SERVICE_REQUESTS" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SERVICE_REQUESTS" TO "SDT_USER";
+GRANT UPDATE ON "SDT_OWNER"."SERVICE_REQUESTS" TO "SDT_USER";
+GRANT DELETE ON "SDT_OWNER"."TARGET_APPLICATIONS" TO "SDT_USER";
+GRANT INSERT ON "SDT_OWNER"."TARGET_APPLICATIONS" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."TARGET_APPLICATIONS" TO "SDT_USER";
+GRANT UPDATE ON "SDT_OWNER"."TARGET_APPLICATIONS" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."BULK_CUST_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."BULK_CUST_APP_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."BULK_SUB_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."ERR_LOG_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."ERR_MESG_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."GLB_PAR_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."IND_REQ_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SER_ROU_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SER_TYP_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."TAR_APP_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SDT_REF_SEQ" TO "SDT_USER";
+GRANT SELECT ON "SDT_OWNER"."SRV_REQ_SEQ" TO "SDT_USER";
+-----------------
+--- System Grants
+-----------------
+--GRANT UNLIMITED TABLESPACE TO "SDT_OWNER";
+--GRANT CREATE SYNONYM TO "SDT_USER";
+--GRANT CREATE SYNONYM TO "SDT_BATCH_USER";
+-----------------
+--- Sys Grants
+-----------------
+--GRANT EXECUTE ON "SYS"."UTL_FILE" TO "SDT_OWNER";
+
+------------------------------------------------
+-- Create Views (not Synonyms!) for SDT_USER
+------------------------------------------------
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUSTOMERS" AS SELECT * FROM "SDT_OWNER"."BULK_CUSTOMERS";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUSTOMER_APPLICATIONS" AS SELECT * FROM "SDT_OWNER"."BULK_CUSTOMER_APPLICATIONS";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUST_APP_SEQ" AS SELECT * FROM "SDT_OWNER"."BULK_CUST_APP_SEQ";
```

Comment:
Does this work for sequences?

Reply:
No

Reply:
If we can't define views for sequences are we going to make any changes so that they can be accessed?

Reply:
Ah! I thought you meant the OR REPLACE

Reply:
We just followed the script and create Views in place of Synonyms where it was declared. How do we know which sequences to make accessible to the two users?